tests/integration-tests/tests/common/utils.py

```diff
@@ -155,3 +156,22 @@ def generate_random_string():
     Example: 4htvo26lchkqeho1
     """
     return "".join(random.choice(string.ascii_lowercase + string.digits) for _ in range(16))  # nosec
+
+
+def reboot_head_node(cluster):
+    logging.info(f"Rebooting head node for cluster: {cluster.name}")
+    username = get_username_for_os(cluster.os)
+    remote_command_executor = RemoteCommandExecutor(cluster, username=username)
```

Comment:
minor: since the RemoteCommandExecutor class maintains an open connection, it would be nice to have an optional arg here to pass in an already-instantiated executor.